server/apps/distributed-app/docker-compose-crowdsec.yml

```diff
@@ -0,0 +1,113 @@
+version: '3'
+
+services:
+
+  james:
+    depends_on:
+      cassandra:
+        condition: service_healthy
+      opensearch:
+        condition: service_started
+      rabbitmq:
+        condition: service_started
+      s3:
+        condition: service_started
+      pulsar:
+        condition: service_started
+    image: apache/james:distributed-latest
```

Comment:
James memory + CrowdSec should be more lightweight for a POC. Anyway not important.